include/zephyr/drivers/timer/nrf_grtc_timer.h

```diff
@@ -101,11 +101,13 @@ void z_nrf_grtc_timer_compare_int_unlock(int32_t chan, bool key);
  *
  * @param chan Channel ID.
  *
```

Comment:
nit:
```suggestion
```

Reply:
Such additional empty lines are present in documentation of other functions in this file too. If you don't mind, I think it's better to stay consistent